docs/asyncapi.yaml

```diff
@@ -127,42 +138,57 @@ components:
         expectedPaymentDate: "21.11.2025"
         adminName: "Jessica Lrrr"
       default: {}
+      minleght: 0
+      maxlenght: 100
 
     reference:
       type: string
-      description: Unique identifier (UUID) for further operations like status query
-      example: "abc123-uuid-123456"
+      description: Unique identifier for further operations like status query
+      example: "can-2a88-be-b855-a55026844MIX"
+      minleght: 2
+      maxlenght: 100
 
     oneclickunsubsurl:
       type: string
       description: URL for one-click unsubscribe option
       example: "https://unsubscribe.example.com"
+      minleght: 4
```

Comment:
urls are always going to be longer than 4

Reply:
Do we care on the length of an URL?  Does Notify have a limit we should align to?

Reply:
4 is a random guard value, Notify doesnt have any limit in their docs, but should we put a guard anyways? we definitely wouldnt want to accept 2000+ symbols as unsubscribe url. Do we want to invalidate this at the receiving stage, rather than allowing it to get passed to notify and other services?

Reply:
Why don't we want to accept 2000?  2000 is still a valid URL so what's our reason for not permitting URLs this long?